### DISCUSSION TITLE
How to enable Bootstrap Containers?

### DISCUSSION THREAD

**Discussion by rgaiacs** [quarto-dev/quarto-cli#5207] [Q&A] [answered]
Similar to https://github.com/quarto-dev/quarto-cli/discussions/2841 but asking because of Bootstrap  variable `enable-container-classes` introduced in https://github.com/twbs/bootstrap/pull/35005.

Adding

```
/*-- scss:defaults --*/

$enable-container-classes: true;
```

to `theme` as in

```
format:
  html:
    theme:
      - _sass/gesis-container.scss
      - cosmo
    css: styles.css
    toc: true
```

still doesn't generate Bootstrap Containers classes.

## How to Reproduce

1. Download [minimal working example](https://github.com/quarto-dev/quarto-cli/files/11247525/bootstrap-containers.zip) and unzip it
2. Change directory to where the files where unzipped
3. Run `quarto preview --no-browser --render html`

## Observed Output

Bootstrap Containers classes is missing.

![Screenshot 2023-04-17 094834](https://user-images.githubusercontent.com/1506457/232419738-4c1697d8-b402-4e57-8017-ff8c001649b7.png)

## Expected Output

Bootstrap Containers classes is present.

## Environment

Output of `quarto check`

```
[✓] Checking versions of quarto binary dependencies...
      Pandoc version 3.1.1: OK
      Dart Sass version 1.55.0: OK
[✓] Checking versions of quarto dependencies......OK
[✓] Checking Quarto installation......OK
      Version: 1.3.324
      Path: /opt/quarto/bin

[✓] Checking basic markdown render....OK

[✓] Checking Python 3 installation....OK
      Version: 3.10.6
      Path: /usr/bin/python3
      Jupyter: 5.2.0
      Kernels: python3

(\) Checking Jupyter engine render....Traceback (most recent call last):
  File "/opt/quarto/share/jupyter/jupyter.py", line 21, in <module>
    from notebook import notebook_execute, RestartKernel
  File "/opt/quarto/share/jupyter/notebook.py", line 18, in <module>
    from nbclient import NotebookClient
ModuleNotFoundError: No module named 'nbclient'
[✓] Checking Jupyter engine render....OK
```

**Comment by cderv** [comment]
The feature you mentioned has been available in Bootstrap 5.2.0 if I am correct. Currently Quarto 1.3 is still on Bootstrap 5.1.3 - we'll probably update for 1.4. 

For now, this is not a feature you can access within Quarto.

@dragonstyle do you know if / when we'll update Bootstrap ?

**Reply by dragonstyle** [reply]
This is a good thing for us to tackle early in the 1.4 release...

**Reply by cderv** [reply, marked as answer]
Cool ! 

Tracked now at https://github.com/quarto-dev/quarto-cli/issues/5210 

Thanks @rgaiacs
